src/pyciemss/Ensemble/interfaces.py

```diff
@@ -102,11 +104,16 @@ def load_and_sample_petri_ensemble(
             - True output a visual
             - False do not output a visual
             - dict output a visual with the dictionary passed to the visualization as kwargs
+        alpha_qs: Optional[Iterable[float]]
+            - The quantiles required for estimating weighted interval score to test ensemble forecasting accuracy.
+        stacking_order: Optional[str]
+            - The stacking order requested for the ensemble quantiles to keep the selected quantity together for each state.
+            - Options: "timepoints" or "quantiles"
 
     Returns:
         samples:
```

Comment:
Can you split this out into two indented values. `samples` and `q_ensemble`, each with their own description as a list?

Reply:
This is the part I am not sure about whether this is the best option for outputs. Should both return statements be a dictionary instead?

Reply:
Also the return type is not consistent